docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -82,7 +82,7 @@
     "### Log the model with training data\n",
     "\n",
     "See the parameter descriptions in {py:meth}`~mlrun.projects.MlrunProject.log_model`. \n",
-    "{Download the pickle file}`pickle file <./src/model2.pkl>` used in this example."
+    "Download the {download}`pickle file <./src/model.pkl>` used in this example."
```

Comment:
There's another pkl file for a different demo - hence the name model2

Reply:
Looking back, there was another in a transient version of a tutorial. So name change is good.